lib/_http_outgoing.js

```diff
@@ -701,7 +704,7 @@ function onError(msg, err, callback) {
 
 function emitErrorNt(msg, err, callback) {
   callback(err);
-  if (typeof msg.emit === 'function' && !msg._closed) {
+  if (typeof msg.emit === 'function') {
```

Comment:
This change is incorrect.

Reply:
> This change is incorrect.

Sorry, I committed the code for the local debug, I'm rolling back the code now